packages/tables/resources/views/index.blade.php

```diff
@@ -464,4 +464,96 @@ class="hidden"
+                            if (line !== null) {
+                                return line;
+                            }
+                        }
+                    }
+
+                    function extractFromString(part, number) {
+                        const matches = part.match(/^[\{\[]([^\[\]\{\}]*)[\}\]](.*)/s);
+
+                        if (matches === null || matches.length !== 3) {
+                            return null;
+                        }
+
+                        var condition = matches[1];
+
+                        var value = matches[2];
+
+                        if (condition.includes(",")) {
+                            const [from, to] = condition.split(",", 2);
+
+                            if (to === "*" && number >= from) {
+                                return value;
+                            } else if (from === "*" && number <= to) {
+                                return value;
+                            } else if (number >= from && number <= to) {
+                                return value;
+                            }
+                        }
+
+                        return condition == number ? value : null;
+                    }
+
+                    function ucfirst(string) {
+                        return string.toString().charAt(0).toUpperCase() + string.toString().slice(1);
+                    }
+
+                    function replace(line, replace) {
+
+                        if (replace.length === 0) {
+                            return line;
+                        }
+
+                        const shouldReplace = {};
+
+                        for (let [key, value] of Object.entries(replace)) {
+                            shouldReplace[":" + ucfirst(key ?? "")] = ucfirst(value ?? "");
+                            shouldReplace[":" + key.toUpperCase()] = value.toString().toUpperCase();
+                            shouldReplace[":" + key] = value;
+                        }
+
+                        Object.entries(shouldReplace).forEach(([key, value]) => {
+                            line = line.replaceAll(key, value);
+                        });
+
+                        return line;
+                    }
+
+                    function stripConditions(segments) {
+                        return segments.map(part => part.replace(/^[\{\[]([^\[\]\{\}]*)[\}\]]/, ''));
+                    }
+
+                    let segments = text.split("|");
+
+                    const value = extract(segments, number);
+
+                    if (value !== null && value !== undefined) {
+                        return replace(value.trim(), variables);
+                    }
+
+                    segments = stripConditions(segments);
+
+                    if (segments.length > 1) {
+                        return replace(number > 1 ? segments[1] : segments[0], variables);
+                    } else {
```

Comment:
`else` is redundant given that both blocks return.

Reply:
Removed else block in latest commit.